internal/testutil/auth.go

```diff
@@ -1,93 +1,22 @@
 package testutil
 
 import (
-	"context"
 	"fmt"
 	"net"
 	"net/http"
 
-	"github.com/golang-jwt/jwt/v4"
 	oauth2 "github.com/oxisto/oauth2go"
 	"github.com/oxisto/oauth2go/login"
 	"golang.org/x/oauth2/clientcredentials"
-	"google.golang.org/grpc/metadata"
 )
 
 const (
 	TestAuthUser         = "clouditor"
```

Comment:
Maybe move the consts also to testdata?

Reply:
Funnily enough, it was already defined there, I removed the constants from this file.